doc/changelog/03-notations/13519-primitiveArrayNotations.rst

```diff
@@ -0,0 +1,9 @@
+- **Added:**
+  :cmd:`Number Notation` and :cmd:`String Notation` can now register parsers
+  and printers for terms containing primitive floats, primitive arrays
+  and type constants for primitive types.
+  (`#13519 <https://github.com/coq/coq/pull/13519>`_,
+  fixes `#13484 <https://github.com/coq/coq/issues/13484>`_
+  and `#13517 <https://github.com/coq/coq/issues/13517>`_,
+  by Fabian Kunze, with help of Jason Gross, reviewed by Gaëtan Gilbert,
+  Pierre Roux, Théo Zimmermann and Jim Fehrle).
```

Comment:
You don't need to credit me as a reviewer.  IMHO reviewer credit is appropriate when the review was a lot of work or "above and beyond" what's normally needed.